docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -52,18 +54,25 @@ be fun and stress free.
 - Several sub-sections describing concrete steps that a user can verify they have completed
 - Wrap up with "Next steps" section.
 
-
 ### How-to guides
 
 **How-to guides** are for readers who already have some familiarity with Teleport and want to set up Teleport for a specific scenario.
 
 #### Expectations
+
 - The introductory section should state the utility of completing the article.
-- Do not break the reader's focus. Do not link the reader to another page—or even another section within the same page—when it is possible to include the same content in the current paragraph or section.
+- Do not recreate content where unnecessary. Actions that are prerequisites to
+  several topics but *not* directly related to those topic should be documented in
+  a single location and referenced in a **Prerequisites** section at the top of the page.
+- Do not break the reader's focus. Do not link the reader to another page—or
+  even another section within the same page—when it is possible to include the
+  same content in the current paragraph or section.
```

Comment:
While we're on the subject: what about content that covers a third party tool we need to partially document in order to proceed? This came up recently with a guide that required a plugin for Fluentd to connect to Datadog, where I provided just enough documentation for the demo setup in the guide, but link to the plugin README for more details. In this case, we don't want to be responsible for documenting third party software beyond what's strictly necessary, but readers following the guide for their custom environment might need the additional content provided by the link.

Reply:
I think it's okay to include a link if there's a clear use for it that brings the reader back to the docs page. E.g., "Read this page for more information" doesn't help a reader get to the end of a how-to guide. "Choose an installation method from this page" does, since the reader knows to come back to the guide once they've installed some software.

Reply:
Agreed, now we just need to figure out how to say that.